res/controllers/Roland_DJ-505-scripts.js

```diff
@@ -1463,8 +1462,8 @@ DJ505.RollMode = function(deck, offset) {
     this.minSize = 0.03125;  // 1/32
     this.maxSize = 32;
 
-    var loopSize;
-    var i;
+    let loopSize;
+    let i;
     for (i = 0; i <= 3; i++) {
         loopSize = (this.loopSize * Math.pow(2, i));
```

Comment:
can't this both be moved into the loop? I don't see where it is used some other place in the function.
```suggestion
    for (let i = 0; i <= 3; i++) {
        const loopSize = (this.loopSize * Math.pow(2, i));
```